quic/s2n-quic-core/src/recovery/cubic.rs

```diff
@@ -757,7 +761,12 @@ impl congestion_controller::Endpoint for Endpoint {
         &mut self,
         path_info: congestion_controller::PathInfo,
     ) -> Self::CongestionController {
-        CubicCongestionController::new(path_info.max_datagram_size)
+        let use_hystart: bool = match env::var(USE_HYSTART) {
+            Ok(_val) => true,
+            Err(_e) => false,
+        };
```

Comment:
We should ideally cache the result of this in a lazy static variable since it's going to be called quite a bit in its current form. If we move it up to the provider level, you could just store the result in a field.

Reply:
You can also use [once_cell](https://docs.rs/once_cell/1.10.0/once_cell/). You'll need to add a dependency to s2n-quic-core Cargo.toml:
```
once_cell = { version = "1", default-features = false}
```

and use it something like:
```rust
use once_cell::sync::OnceCell;
static USE_HYSTART_PP: OnceCell<bool> = OnceCell::new();
#[cfg(feature = "std")]
let use_hystart_plus_plus = *USE_HYSTART_PP.get_or_init(std::env::var(USE_HYSTART_PLUS_PLUS).is_ok());
```

I'm not sure its worth moving up to the provider, since this is temporary anyway

Reply:
This part has been updated in hybrid_slow_start.rs like this. 
https://github.com/nsdyoshi/s2n-quic/blob/hystartplus/quic/s2n-quic-core/src/recovery/hybrid_slow_start.rs#L78-L81

Do you think this is enough?

Reply:
Got it. I updated the code.